Source/SwiftLintFramework/Rules/RuleConfigs/NameConfig.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  NameConfig.swift
+//  SwiftLint
+//
+//  Created by Scott Hoyt on 1/19/16.
+//  Copyright © 2016 Realm. All rights reserved.
+//
+
+import Foundation
+
+public struct NameConfig: RuleConfig, Equatable {
+    var minLength: SeverityLevelsConfig
+    var maxLength: SeverityLevelsConfig
+    var excluded: Set<String>
+
+    var minLengthThreshold: Int {
+        return Swift.max(minLength.warning, minLength.error)
```

Comment:
I'll change that. This was a leftover from when I originally had the `minLength` and `maxLength` properties shortened to `min` and `max` so I needed to namespace those calls.